common/src/main/java/com/wynntils/utils/objects/ThrowingBiPredicate.java

```diff
@@ -5,7 +5,6 @@
 package com.wynntils.utils.objects;
 
 @FunctionalInterface
-public interface ThrowingBiPredicate<T, U, Exception extends Throwable> {
-
-    boolean test(T t, U u) throws Exception;
+public interface ThrowingBiPredicate<T, U, EXC extends Throwable> {
```

Comment:
Actually just use `E`. The convention is to use single-letter for type templates in Java.